### PR TITLE
cxf-rt-rs-client: set Accept header to wildcard for void response

### PR DESCRIPTION
**Pull request by amarkevich**

### CODE REVIEW

File: rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/ClientProxyImpl.java [hunk 1]
```diff
@@ -477,16 +477,18 @@ protected void checkResponse(Method m, Response r, Message inMessage) throws Thr
         List<MediaType> accepts = getAccept(headers);
         if (accepts == null) {
             List<MediaType> produceTypes = ori.getProduceTypes();
```
Comment (coheigea) [inline rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/ClientProxyImpl.java]: LGTM but here "produceTypes" could be in the else statement instead.

**Comment by mlschechter** [comment]
@coheigea - Can you please let me know when this will be released? We just implemented a media type whitelisting feature, and the bug this fixes is stopping us from deploying.

Thanks!

**Comment by coheigea** [comment]
Realistically with the holiday season - the second week of January I'd say.

**Comment by mlschechter** [comment]
That's great - thanks!

**Comment by mlschechter** [comment]
Thanks for the release, @coheigea - everything's working great!

**Pull request merged**